DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tc_flower_utils.py

```diff
@@ -347,7 +348,7 @@ def random_icmp_type():
         filter_t['vlan_id'] = random.randint(1, 4095)
         if want_ip:
             filter_t['vlan_ethtype'] = random.choice(ip_protocols)
```

Comment:
make sure that you don't override vlan_ethtype in case of tagged ip packets
```suggestion
        if want_vlan_ethtype:
            if want_ip:
                filter_t['vlan_ethtype'] = random.choice(ip_protocols)
            else:
                filter_t['vlan_ethtype'] = f'0x9{random.randint(1, 3)}00'
```